spec/mailers/onboarding_mailer_spec.rb

```diff
@@ -0,0 +1,115 @@
+require 'rails_helper'
+
+RSpec.describe OnboardingMailer do
+  let(:school){ create(:school, name: 'Test School') }
+  let(:user){ create(:onboarding_user, school: school) }
+  let(:school_onboarding) { create(:school_onboarding, school_name: 'Test School', created_by: user, school: school) }
+
+  describe '#onboarding_email' do
+    it 'sends the onboarding email' do
+      OnboardingMailer.with(emails: ['test@blah.com'], school_onboarding: school_onboarding).onboarding_email.deliver_now
+      email = ActionMailer::Base.deliveries.last
+      expect(email.subject).to eq("Set up your school on Energy Sparks")
+      I18n.t('onboarding_mailer.onboarding_email').except(:subject).values.each do |email_content|
+        expect(email.body.to_s).to include(email_content.gsub('%{school_name}', 'Test School'))
+      end
+    end
+  end
+
+  describe '#completion_email' do
+    it 'sends the completion email' do
+      OnboardingMailer.with(emails: ['test@blah.com'], school_onboarding: school_onboarding).completion_email.deliver_now
+      email = ActionMailer::Base.deliveries.last
+      expect(email.subject).to eq("Test School has completed the onboarding process")
```

Comment:
Just wondered if you could use the subject from the yaml file here instead - means you don't have to change the test later if the yaml changes?